daemon/tests/harness/mod.rs

```diff
@@ -365,17 +364,6 @@ macro_rules! deliver_event {
     };
 }
 
-async fn in_memory_db() -> SqlitePool {
-    // Note: Every :memory: database is distinct from every other. So, opening two database
-    // connections each with the filename ":memory:" will create two independent in-memory
-    // databases. see: https://www.sqlite.org/inmemorydb.html
```

Comment:
nit: any reason for getting rid of this comment? I for one found it useful (and it doesn't look like it can/will rot)

Reply:
No reason, just got lost in the refactor!